src/fpm_backend.f90

```diff
@@ -266,29 +267,34 @@ subroutine build_target(model,target,stat)
     select case(target%target_type)
 
     case (FPM_TARGET_OBJECT)
+        write(*,('(a)')) esc('<E><bo><g> compile:')//' '//target%source%file_name
         call run(model%fortran_compiler//" -c " // target%source%file_name // target%compile_flags &
-              // " -o " // target%output_file, echo=.true., exitstat=stat)
+              // " -o " // target%output_file, echo=.false., exitstat=stat)
 
     case (FPM_TARGET_C_OBJECT)
+        write(*,('(a)')) esc('<E><bo><g> compile:')//' '//target%source%file_name
         call run(model%c_compiler//" -c " // target%source%file_name // target%compile_flags &
-                // " -o " // target%output_file, echo=.true., exitstat=stat)
+                // " -o " // target%output_file, echo=.false., exitstat=stat)
 
     case (FPM_TARGET_EXECUTABLE)
 
+        write(*,('(a)')) esc('<E><bo><c>    load:')//' '//basename(target%output_file)
```

Comment:
```suggestion
        write(*,('(a)')) esc('<E><bo><c>    link:')//' '//basename(target%output_file)
```

Very minor point but I think these days it is much more common to refer to __linking__ of symbols rather than __loading__ them.